pkg/controller/egress/controller_test.go

```diff
@@ -17,6 +17,7 @@ package egress
 import (
```

Comment:
I scanned the new test cases quickly. Do we have one for an egressIPPool with multiple different IP ranges?

Reply:
I added multiple IPRanges cases in ipallocator's unit tests. In latest patch, I just added one in this file too. See "Egress with empty EgressIP and existing ExternalIPPool"